conpot/protocols/kamstrup/meter_protocol/kamstrup_server.py

```diff
@@ -110,16 +107,3 @@ def start(self, host, port):
 
     def stop(self):
         self.server.stop()
-
```

Comment:
This doesn't impact the implementation. Some devs use it to debug the protocol locally. Super helpful. I think we can leave it here.

Reply:
These bits were refactored into `start_protocol.py` in 368996404b6bdce5035afa96e89490a1fd0a3eca. The 
next commit c4dad04f610522fd1d81bd9ff2e26b602b398a62 then uses that groundwork to test starting and stopping all protocols individually.

Some other changes in this MR would have made it necessary to rewrite these sections. I tried to do that in only one location that was used by both tests and `start_protocol.py`.

I can reinstate the separate sections (albeit rewritten) in lieu of `start_protocol.py` if you want.

Reply:
as in keeping the `__main__` section and using `start_protocol` for initiating debugging?

Reply:
Just to add here: with `__main__` you can just go the file itself and starting the protocol with something like `python3 <protcol_name>.py` Again it's just useful for debugging the protocol locally.

It is not used anywhere for running the protocols which is done `conpot` executable. So totally in favour of keeping the `__main__` sections.

Reply:
`start_protocol.py` was added for exactly this purpose. It accepts a single command line parameter which identifies the protocol, i. e. `start_protocol.py <protocol_name>`.

Thus, instead of
```sh
python3 kamstrup_server.py
```
you do
```sh
python3 start_protocol.py kamstrup_meter
```

Advantages are that the startup code is consistently the same, i.e. the protocols are run the same way the tests and (more or less) bin/conpot run them, and that the code is in one place rather than scattered throughout.

Disadvantages are that the code is not as close to the protocols as before and that if you want to play around with the startup logic of one protocol it affects all of them. (The latter may be an advantage, given that you want consistency and all.)

I can remove `start_protocols.py` and restore the `__main__` sections or keep it as is.